### ISSUE TITLE
[Feature Request] Deleting Projects

### ISSUE THREAD

**Issue by DxCx** [graphcool/graphcool-cli#80]
Hey,

I would like to be able to delete an existing project =)

**Comment by schickling** [comment]
This is now possible by running `graphcool delete` or `graphcool delete <project-id>`

**Comment by wordyallen** [comment]
I rm'd `graphcool.project` ... now I'm getting this:

 Error: Cannot convert undefined or null to object

is there  place I can delete the reference?

**Comment by schickling** [comment]
We'll look into it @wordyallen. Can you reproduce the problem with another project?

**Comment by artivilla** [comment]
Thought I'd search if someone had the same issue, I tried deleting the example project already provided in graphcool console via the CLI. Here's the log with the keys removed:
```
~/Dropbox/Repos/hackernews
❯ graphcool delete  

❯ ◯  Example Project (xxx)
  ◯  Hackernews (xxx)
  ◯  BarterChat (xxx)

Are you absolutely sure you want to delete these projects? This operation can not be undone! [y|N]

Are you absolutely sure you want to delete these projects? This operation can not be undone! [y|N]
~/Dropbox/Repos/hackernews 6s
❯ graphcool delete
✖  Error: Cannot convert undefined or null to object
```

**Comment by marktani** [comment]
Hey @artivilla @wordyallen, does this also happen to a fresh project? Does this happen for projects across all regions?

@artivilla what do you mean with `create-react-app@1.4.0`? Also, what's your `graphcool -v`?

**Comment by artivilla** [comment]
`create-react-app` helps you scaffold new react applications. I guess at some point in the past I had a hackernews app that I deleted and I'm not creating a new application and wanted to delete the GC project I had already setup in the past. I'm using v1.4.0 for the create-react-app.
```
~/Dropbox/Repos/hackernews
❯ graphcool -v
1.2.1
```

**Comment by marktani** [comment]
I don't see how `create-react-app` plays into this issue at all. Wait, are you actually running `graphcool delete` twice in a row, and the error only occurs at the second time?

**Comment by artivilla** [comment]
I tried doing it few times b/c i opened the GC console and the project still exists. Sorry it has nothing to with create-react-app. Here's the debugger turned on:
```
~/Dropbox/Repos/hackernews
❯ graphcool delete      
  graphcool Send request to https://api.graph.cool/system
  graphcool Payload: 
  graphcool {"query":"{\n  viewer {\n    user {\n      projects {\n        edges {\n          node {\n            id\n            name\n            schema\n            alias\n            version\n            region\n          }\n        }\n      }\n    }\n  }\n}","variables":{}} +0ms
  graphcool Received JSON response: 
  graphcool {"data":{"viewer":{"user":{"projects":{"edges":[{"node":{"name":"Example Project","alias":null,"version":1,"id":"cj5hj9jawbckq012200ct86ej","schema":"type File implements Node {\n  contentType: String!\n  createdAt: DateTime!\n  id: ID! @isUnique\n  name: String!\n  secret: String! @isUnique\n  size: Int!\n  updatedAt: DateTime!\n  url: String! @isUnique\n}\n\ntype User implements Node {\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}","region":"EU_WEST_1"}},{"node":{"name":"BarterChat","alias":null,"version":6,"id":"cj5q8fidumaek0122vj5xkeyd","schema":"type Customer implements Node {\n  name: String!\n  conversations: [Conversation!]! @relation(name: \"ConversationsFromCustomer\")\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}\n\ntype Message implements Node {\n  text: String!\n  agent: Agent @relation(name: \"MessagesFromAgents\")\n  conversation: Conversation @relation(name: \"MessagesInConversation\")\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}\n\ntype Agent implements Node {\n  slackUserId: String!\n  slackUserName: String!\n  conversations: [Conversation!]! @relation(name: \"ConversationsFromAgent\")\n  messages: [Message!]! @relation(name: \"MessagesFromAgents\")\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}\n\ntype Conversation implements Node {\n  slackChannelIndex: Int!\n  customer: Customer @relation(name: \"ConversationsFromCustomer\")\n  agent: Agent @relation(name: \"ConversationsFromAgent\")\n  messages: [Message!]! @relation(name: \"MessagesInConversation\")\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}\n\ntype File implements Node {\n  contentType: String!\n  createdAt: DateTime!\n  id: ID! @isUnique\n  name: String!\n  secret: String! @isUnique\n  size: Int!\n  updatedAt: DateTime!\n  url: String! @isUnique\n}\n\ntype User implements Node {\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n  username: String!\n  firstName: String!\n  lastName: String!\n  emailAddress: String!\n}","region":"US_WEST_2"}},{"node":{"name":"Hackernews","alias":null,"version":3,"id":"cj6nalb12111j0143wt8ywj84","schema":"type Link implements Node {\n  url: String!\n  description: String!\n  createdAt: DateTime!\n  id: ID! @isUnique\n  name: String\n  updatedAt: DateTime!\n}\n\ntype File implements Node {\n  contentType: String!\n  createdAt: DateTime!\n  id: ID! @isUnique\n  name: String!\n  secret: String! @isUnique\n  size: Int!\n  updatedAt: DateTime!\n  url: String! @isUnique\n}\n\ntype User implements Node {\n  createdAt: DateTime!\n  id: ID! @isUnique\n  updatedAt: DateTime!\n}","region":"US_WEST_2"}}]}}}}} +942ms

❯ ◯  Example Project (xxx)
  ◯  BarterChat (xxx)
  ◯  Hackernews (xxx)
⠋ Deleting 0 projects ...  graphcool Send request to https://api.graph.cool/system
  graphcool Payload: 
  graphcool {"query":"mutation ()  { \n \n}","variables":{}} +9s
⠸ Deleting 0 projects ...  graphcool Received JSON response: 
  graphcool {"error":"Syntax error while parsing GraphQL query. Invalid input \"mutation ()\", expected OperationDefinition, FragmentDefinition or TypeSystemDefinition (line 1, column 1):\nmutation ()  { \n^"} +268ms
✖  Error: Cannot convert undefined or null to object
  graphcool TypeError: Cannot convert undefined or null to object
  graphcool     at Function.keys (<anonymous>)
  graphcool     at Object.<anonymous> (/Users/artichandok/.nvm/versions/node/v7.9.0/lib/node_modules/graphcool/dist/src/api/api.js:281:35)
  graphcool     at Generator.next (<anonymous>)
  graphcool     at fulfilled (/Users/artichandok/.nvm/versions/node/v7.9.0/lib/node_modules/graphcool/dist/src/api/api.js:4:58)
  graphcool     at process._tickDomainCallback (internal/process/next_tick.js:135:7) +3ms
```
I just tried it using the -p flag and it works. Seems like the interactive mode didn't work for me.

**Comment by kbrandwijk** [comment]
Seems to be fixed.

**Comment by timsuchanek** [comment]
Fixed in 1.3.13 beta
